common/plugins/add_endpoint.go

```diff
@@ -135,7 +136,7 @@ func SetupVethWithNames(lxcIfName, tmpIfName string, mtu int, ep *models.Endpoin
 		return nil, nil, fmt.Errorf("unable to lookup veth just created: %s", err)
 	}
 
-	if err = netlink.LinkSetMTU(hostVeth, mtu); err != nil {
```

Comment:
@joestringer we allow the MTU to be changed in the CNI configuration. Since the CNI configuration is read for every spawn container it means we will ignore the most recent value stored in the configuration file. And, it seems the user can't set the MTU with a flag?

Reply:
@aanm correct, it won't sync with that. I can add a note about this. In future if we wanted to handle this nicely then we could add a config option to the API and update all containers to have the same standard MTU whenever a container joins with a lower MTU. For now, we just rely on users using default values or knowing what they're doing.

Reply:
I updated the description above `StandardMTU` to describe its effects between different packages. At this stage I don't plan on adding a flag for MTU.